content/concepts/dolthub/permissions.md

```diff
@@ -2,4 +2,34 @@
 title: Permissions
 ---
 
-# Permissions
\ No newline at end of file
+# Permissions
+
+It's important that you have full control over can access or change your data. There are a few ways to control users' roles for your database on DoltHub.
+
+## Read permissions
```

Comment:
I think you should talk about clone and push a bit here. Like what exactly qualifies as a read and a write.